appsdk/configurable.go

```diff
@@ -359,3 +329,74 @@ func (dynamic AppFunctionsSDKConfigurable) SetOutputData() appcontext.AppFunctio
 	transform := transforms.OutputData{}
 	return transform.SetOutputData
 }
+
+// BatchByCount ...
+func (dynamic AppFunctionsSDKConfigurable) BatchByCount(parameters map[string]string) appcontext.AppFunction {
+	batchThreshold, ok := parameters[BatchThreshold]
+	if !ok {
+		dynamic.Sdk.LoggingClient.Error("Could not find " + BatchThreshold)
+		return nil
+	}
+
+	thresholdValue, err := strconv.Atoi(batchThreshold)
+	if err != nil {
+		dynamic.Sdk.LoggingClient.Error(fmt.Sprintf("Could not parse '%s' to an int for '%s' parameter", batchThreshold, BatchThreshold), "error", err)
+		return nil
+	}
+	transform, err := transforms.NewBatchByCount(thresholdValue)
+	if err != nil {
+		dynamic.Sdk.LoggingClient.Error(err.Error())
+	}
+	dynamic.Sdk.LoggingClient.Debug("Batch by count Parameters", batchThreshold)
```

Comment:
Logging client expects key/value pairs after the initial message. Not sure how it will handle this. Normally I would put  `("Batch by count Parameters", BatchThreshold, batchThreshold)`